dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/src/org/apache/cxf/jaxrs/utils/JAXRSUtils.java

```diff
@@ -2169,16 +2169,18 @@ private static String combineUriTemplates(final String parent, final String chil
         // The way URI templates are normalized in org.apache.cxf.jaxrs.model.URITemplate:
         //  - empty or null become "/"
         //  - "/" is added at the start if not present 
-        if ("/".equals(parent)) {
+        //Liberty Change start
+        if (PATH_SEGMENT_SEP.equals(parent)) {
```

Comment:
In the end the byte code is the same whether you use "/" or PATH_SEGMENT_SEP.  So not sure if it is worth to make the change.  If you leave those the way they were, then you only need the change flags on the one line where you changed replaceAll with substring.